tests/Integration/Behaviour/Features/Scenario/Category/category_management.feature

```diff
@@ -0,0 +1,25 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s category
+@reset-database-before-feature
+Feature: Category Management
+  PrestaShop allows BO users to manage categories for products
+  As a BO user
+  I must be able to create, edit and delete categories in my shop
+
+  Scenario: Adding new Category
+    Given I specify "name" "PC parts" for new category "category1"
+    And I specify displayed to be "enabled" for new category "category1"
+    And I specify "parent category" "Home Accessories" for new category "category1"
+    And I specify "description" "Best PC parts" for new category "category1"
+    And I specify "meta title" "PC parts meta title" for new category "category1"
+    And I specify "meta description" "PC parts meta description" for new category "category1"
+    And I specify "friendly url" "pc-parts" for new category "category1"
+    And I specify group access for "Customer,Guest,Visitor" for new category "category1"
```

Comment:
@matks instead of using table I've decided to write step for each property? The amount of lines are the same, but the context that it gives (for example compare `And I specify group access` with `And I specify "meta title"`, in case of table, they would be the same) is much clearer imo, wdyt?

Reply:
It works 😄 I find tables more usable and more readable but I find them especially useful to avoid writing all these steps. If writing all these steps is not bothering you too much, all right.